wiki/Article_styling_criteria/Formatting/en.md

```diff
@@ -314,6 +314,23 @@ Section headings must not skip a heading level (i.e. do not go from a level 2 he
 
 *Notice: On the website, heading levels 4 and 5 will not appear in the table of contents. They cannot be linked to directly either.*
 
+### Custom identifiers
+
+It is possible to redefine a section's identifier, which is used for linking to it directly:
+
+```markdown
+## My cooldown has passed. How do I appeal? {#appeal}
+
+## Common restriction reasons and cooldowns {#common-reasons}
+```
+
+Custom identifiers should be used in case the automatically generated ones are too long, contain tricky punctuation marks or images. This feature can also be used for tagging a specific part of the article which doesn't have a heading. Use it sparingly:
```

Comment:
```suggestion
Custom identifiers should be used in case the automatically generated ones are too long, contain tricky punctuation marks or images. 

This feature can also be used for tagging a specific part of the article which doesn't have a heading. Use it sparingly:
```

Reply:
yes, you are right. in this case I'd rather attach the first sentence to the one before the example